# HG changeset patch
# User Vytautas Janƒçauskas <unaudio@gmail.com>
# Date 1379937867 -10800
#      Mon Sep 23 15:04:27 2013 +0300
# Node ID 9c56d3fafb9e2834641db8e6b1540e19230efc4a
# Parent  0ac6a4e7369b46697d156ccaa8a732ac9c650baa
maint: removed audiodevinfo.txi, audiofile.txi, audiorecorder.txi and audioplayer.txi files because they are no longer needed

diff --git a/doc/interpreter/audiodevinfo.txi b/doc/interpreter/audiodevinfo.txi
deleted file mode 100644
--- a/doc/interpreter/audiodevinfo.txi
+++ /dev/null
@@ -1,6 +0,0 @@
-@node Audio Device Information
-@section Audio Device Information
-
-This function is used to get information about audio devices that are available on your system. This information can then be used when creating audioplayer and audiorecorder objects.
-
-@DOCSTRING(audiodevinfo)
\ No newline at end of file
diff --git a/doc/interpreter/audiofile.txi b/doc/interpreter/audiofile.txi
deleted file mode 100644
--- a/doc/interpreter/audiofile.txi
+++ /dev/null
@@ -1,11 +0,0 @@
-@node Audio File Utilities
-@section Audio File Utilities
-
-The following functions provide functionality for reading and writing audio files in various formats including .wav, .flag, .ogg and many others. Also for retrieving information about audio files, such as number of channels, sampling rate, metadata and other.
-
-@DOCSTRING(audioinfo)
-
-The following two functions read and write audio files.
-
-@DOCSTRING(audioread)
-@DOCSTRING(audiowrite)
\ No newline at end of file
diff --git a/doc/interpreter/audioplayer.txi b/doc/interpreter/audioplayer.txi
deleted file mode 100644
--- a/doc/interpreter/audioplayer.txi
+++ /dev/null
@@ -1,5 +0,0 @@
-@node Audio Player
-@section Audio Player
-
-@DOCSTRING(@@audioplayer/audioplayer)
-@DOCSTRING(@@audioplayer/play)
\ No newline at end of file
diff --git a/doc/interpreter/audiorecorder.txi b/doc/interpreter/audiorecorder.txi
deleted file mode 100644
--- a/doc/interpreter/audiorecorder.txi
+++ /dev/null
@@ -1,5 +0,0 @@
-@node Audio Recorder
-@section Audio Recorder
-
-@DOCSTRING(@audiorecorder/audiorecorder)
-@DOCSTRING(@audiorecorder/record)
\ No newline at end of file
